src/client/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -110,10 +107,27 @@ public async Task<IEnumerable<IAccount>> GetAccountsAsync()
                         new AcquireTokenCommonParameters(), 
                         requestContext));
 
-                accounts = await cacheSessionManager.GetAccountsAsync(Authority).ConfigureAwait(false);
+                localAccounts = await cacheSessionManager.GetAccountsAsync(Authority).ConfigureAwait(false);
+            }
+
+            if (AppConfig.IsBrokerEnabled && ServiceBundle.PlatformProxy.CanBrokerSupportSilentAuth())
+            {
+                //Broker is available so accounts will be merged using home account ID with Broker taking priority
```

Comment:
confirmed with android that they prioritize local accounts when there are duplicates in the broker and local caches. once the local tokens expire, MSAL will use the broker account instead. We can do the same here.